crates/router/src/core/payments.rs

```diff
@@ -463,6 +463,9 @@ where
     .await
     .change_context(errors::ApiErrorResponse::InternalServerError)
     .attach_printable("Could not decode the conditional config")?;
+
+    logger::debug!(ConditionalConfigs=?output, "Conditional Config after dsl execute");
```

Comment:
It's not `ConditionalConfig`. That name is something we have to refactor in the codebase. Also prefer snake case for keys in the logger for consistency across application.
```suggestion
    logger::debug!(decision_manager_output=?output, "Decision Manager output after execution");
```